test/resources/ostree-health.resource

```diff
@@ -35,3 +35,8 @@ System Should Be Healthy
 
     ${health}=    Get Persisted System Health
     Should Be Equal As Strings    ${health}    healthy
+
+Restart Greenboot And Wait For Success
+    [Documentation]    Restart the greenboot-healthcheck service and check its status
+    Systemctl    restart    greenboot-healthcheck.service
+    Wait Until Greenboot Health Check Exited
```

Comment:
This only waits for greenboot to finish, right, without checking the results? But line 39 says "wait for success".

Reply:
The `Greenboot Health Check Exited` keyword is checking the status of the service, which is supposed to be `exited`
See https://github.com/openshift/microshift/pull/2325/files#diff-91546dab3e6e25e6ab48a09f02c8040d107f5cf9f8e40b06cd1babd6cb25b395R20